### PR TITLE
instructions benches

### LINKED ISSUE #1729: BENCHMARKING: i64.const instruction weight
### Problem

https://github.com/gear-tech/gear/blob/4dd6c63ef788fb7343cd6a20445a8268fb9614e0/pallets/gear/src/benchmarking/mod.rs#L1947-L1958
Wasmer singlepass has optimisation, which generates real const operation only in places where it uses, but because of drop in benchmarks this operations won't be generated. Also in benchmarks const instruction has small literal. But in fact operations with large literals will be generated in several host native operations, which also increase their execution time.

See, also https://github.com/gear-tech/gear/issues/1382

### PR DESCRIPTION
**Pull request by grishasobol**
Fix #1729 
Calculate now  const weight as: ```call function with i64const weight - call empty function weight```
Change some benchmarks in order to improve weights accuracy.
Re-run benchmarks on `c6id.2xlarge` AWS: `Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz`. See more: https://wiki.polkadot.network/docs/maintain-guides-how-to-validate-polkadot#reference-hardware .

Instructions weights difference for `gear-dev`:
WASM Instruction | New | Old
-- | -- | --
i64const | 301ps | 29ps
i64load | 4.5ns | 7.4ns
i64store | 6.4ns | 12.9ns
select | 7.0ns | 6.0ns
r#if | 5.9ns | 6.2ns
br | 3.2ns | 4.8ns
br_if | 5.9ns | 5.2ns
br_table | 9.9ns | 15.8ns
br_table_per_entry | 375ps | 87ps
call | 4.8ns | 6.5ns
call_indirect | 19.4ns | 14.9ns
call_indirect_per_param | 1.7ns | 729ps
local_get | 543ps | 275ps
local_set | 1.5ns | 550ps
local_tee | 1.5ns | 511ps
global_get | 1.7ns | 427ps
global_set | 2.6ns | 798ps
memory_current | 14.1ns | 12.4ns
i64clz | 6.0ns | 7.6ns
i64ctz | 5.7ns | 7.3ns
i64popcnt | 1.1ns | 423ps
i64eqz | 3.4ns | 993ps
i64extendsi32 | 757ps | 293ps
i64extendui32 | 337ps | 181ps
i32wrapi64 | 300ps | 183ps
i64eq | 3.0ns | 978ps
i64ne | 3.0ns | 968ps
i64lts | 3.0ns | 993ps
i64ltu | 2.8ns | 992ps
i64gts | 3.1ns | 984ps
i64gtu | 3.0ns | 992ps
i64les | 3.0ns | 983ps
i64leu | 3.0ns | 1000ps
i64ges | 3.0ns | 979ps
i64geu | 2.9ns | 992ps
i64add | 2.2ns | 841ps
i64sub | 2.0ns | 813ps
i64mul | 2.9ns | 1.1ns
i64divs | 4.1ns | 4.5ns
i64divu | 4.5ns | 4.5ns
i64rems | 18.3ns | 8.2ns
i64remu | 4.7ns | 4.5ns
i64and | 2.2ns | 790ps
i64or | 2.2ns | 831ps
i64xor | 2.2ns | 812ps
i64shl | 1.8ns | 736ps
i64shrs | 1.8ns | 751ps
i64shru | 1.8ns | 745ps
i64rotl | 1.8ns | 751ps
i64rotr | 1.8ns | 743ps

### CODE REVIEW

File: pallets/gear/src/weights.rs [hunk 1]
```diff
@@ -19,12 +19,12 @@
 //! Autogenerated weights for pallet_gear
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-09-17, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
-//! HOSTNAME: `epyc-runners-node.hetzner`, CPU: `AMD EPYC 7502P 32-Core Processor`
+//! DATE: 2022-11-05, STEPS: `51`, REPEAT: 200, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `ip-172-31-20-193.us-west-1.compute.internal`, CPU: `Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz`
```
Comment (breathx) [inline pallets/gear/src/weights.rs:23]: Is it ok to see here ip of the machine?

**Review by breathx — APPROVED** [review]
Awesome

File: pallets/gear/src/weights.rs (hunk 1 above)
Comment (grishasobol) [reply pallets/gear/src/weights.rs:23]: Hmm, may be it's not ok. Ok,  I will remove this

Comment (grishasobol) [reply pallets/gear/src/weights.rs]: Removed it

**Pull request merged**